examples/flakes/overlay/yarn-overlay/flake.nix

```diff
@@ -3,7 +3,7 @@
     "This flake outputs a modified version of Yarn that uses NodeJS 16";
 
   inputs = {
-    nixpkgs.url = "nixpkgs/nixos-21.11";
+    nixpkgs.url = "nixpkgs/fc3de6da83863f8f36fdcac1c199c6066a6a0378";
```

Comment:
This is a bit fragile, but not sure there another way beyond hardcoding the hash in devbox.json

Reply:
They'll diverge if we delete or update the lockfile, but I'm not sure there's a better way to do it. I think if we had something like devbox.nix we could use the project details in the user flakes